child_compassion/mappings/compassion_child_mapping.py

```diff
@@ -32,6 +32,54 @@ class GenericChildMapping(OnrampMapping):
         'GlobalId': 'global_id',
         'LocalBeneficiaryId': 'local_id',
 
+        'AcademicPerformance': 'academic_performance',
+        'ActiveProgram': 'type',
+        'AgeInYearsAndMonths': None,
+        'BeneficiaryStatus': 'state',
+        'Birthdate': 'estimated_birthdate',
+        'CDSPType': 'cdsp_type',
+        'ChristianActivities': (
+            'christian_activity_ids', 'child.christian.activity'),
+        'ChronicIllness': ('chronic_illness_ids', 'child.chronic.illness'),
+        'CognitiveAgeGroup': 'cognitive_age_group',
+        'CommunityName': ('project_id.community_name''compassion.project'),
+        'CompassBeneficiaryID': 'compass_id',
+        'CorrespondenceLanguage': 'correspondence_language',
+        'CurrentUniversityYearOfStudy': None,
+        'EstimatedBirthDate': 'estimated_birthdate',
+        'ExpectedTransitionDateToSponsorship': None,
+        'FO': ('project_id.name', 'compassion.project'),
+        'FavoriteProjectActivities': ('activity_ids',
+                                      'child.project.activity'),
+        'FavoriteSubjectInSchool': ('subject_ids', 'child.school.subject'),
+        'FormalEducationLevel': 'education_level',
+        'HeightInCm': 'height',
+        'HouseholdDuties': ('duty_ids', 'child.household.duty'),
+        'HouseholdInformation': ('household_id', 'compassion.household'),
+        'HouseholdMembersList': ('household_id.member_ids',
+                                 'compassion.household.member'),
+        'ICP': ('project_id.name', 'compassion.project'),
```

Comment:
In my experience, having two relational fields mapped to the same object (in this case `project_id`) will cause a problem to find the correct project for the child. I would advise to remove field `ICP` and only keep `ICPID` to find the right project. We already know the project name and thus don't need to read that value.